main.js

```diff
@@ -13,10 +13,10 @@ var ideas = [];
 //Event listeners
 saveButton.addEventListener("click", saveIdea);
 ideaForm.addEventListener("change", enableSaveButton);
-ideaBox.addEventListener("click", deleteMiniIdea);
+ideaBox.addEventListener("click", whatWasClicked);
 starButton.addEventListener("click", changeStar);
```

Comment:
I think we have duplicate starbutton eventlisteners (this one and the one on line 109) we will need to clean up on the next pass through :)